### DISCUSSION TITLE
How do you connect a django server to pywebview?

### DISCUSSION THREAD

**Discussion by byte127x** [r0x0r/pywebview#980] [Q&A]
Hello.
Up until now, I have been using 
" import webview, subprocess

subprocess.Popen(['python', 'manage.py', 'runserver'])
window = webview.create_window('MusicMedia', 'http://127.0.0.1:8000/')

webview.start() "

to run django servers with pywebview.
The problem with this is that the server doesn't die after the pywebview window is closed.
Is there any solution?

**Comment by r0x0r** [comment]
would this do the trick?

```
import webview, subprocess

p = subprocess.Popen(['python', 'manage.py', 'runserver'])
window = webview.create_window('MusicMedia', 'http://127.0.0.1:8000/')

webview.start()
p.terminate()
```
